### PR TITLE
dnsviz update to 0.6.6 (new version)

### PR DESCRIPTION
**Pull request by cdeccio**
###### Tested on
macOS 10.x
Xcode 8.x

###### Verification 
Have you
- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by mojca** [comment]
I believe you are the maintainer. Please add your github handle to the Portfile.

(See maintainers under https://guide.macports.org/#reference.keywords in our slightly outdated guide. You should have both email + github handle there.)

**Comment by cdeccio** [comment]
Thanks - I didn't know that was a thing.  I've committed the change.

File: net/dnsviz/Portfile [hunk 1]
```diff
@@ -5,22 +5,22 @@ PortSystem          1.0
...
-github.setup        dnsviz dnsviz 0.6.5 v
+github.setup        dnsviz dnsviz 0.6.6 v
 github.tarball_from releases
 
 categories          net
 platforms           darwin
 license             GPL-2+
 supported_archs     noarch
-maintainers         deccio.net:casey openmaintainer
+maintainers         @cdeccio deccio.net:casey openmaintainer
```
Comment (cdeccio) [inline net/dnsviz/Portfile]: Thanks.  I've made that change.

**Comment by cdeccio** [comment]
Thanks, much!

**Pull request merged**